airflow/providers/amazon/aws/secrets/systems_manager.py

```diff
@@ -101,6 +105,37 @@ def get_conn_uri(self, conn_id: str) -> Optional[str]:
 
         return self._get_secret(self.connections_prefix, conn_id)
 
+    def get_conn_kwargs(self, conn_id: str) -> Optional[str]:
+        if self.connections_prefix is None:
+            return None
+
+        val = self._get_secret(path_prefix=self.connections_prefix, secret_id=conn_id)
+        if val:
+            kwargs = json.loads(val)
+            extra = kwargs.pop('extra', None)
+            if extra:
+                kwargs['extra'] = extra if isinstance(extra, str) else json.dumps(extra)
+            return kwargs
+
+    def get_connection(self, conn_id: str) -> Optional['Connection']:
+        """
+        Return connection object with a given ``conn_id``.
+
+        :param conn_id: connection id
+        :type conn_id: str
+        """
+        from airflow.models.connection import Connection
```

Comment:
Isn't this already imported at the module level?

Reply:
yup good point, copy paste oversight